app/src/common/urls.js

```diff
@@ -89,6 +89,17 @@ export const URLS = {
     `${urlBase}${activeProject}/launch/attribute/values?${
       key ? `filter.eq.attributeKey=${key}&` : ''
     }filter.cnt.attributeValue=${searchTerm}`,
+  itemAttributeKeysByLaunchName: (activeProject, launchName = '') => (searchTerm = '') =>
+    `${urlBase}${activeProject}/item/step/attribute/keys?${
```

Comment:
Can the `getQueryParams` util be used here?

Reply:
Agree. `getQueryParams` also solves the problem with encryption symbols in URL

Reply:
@AmsterGet @ElenaRomanchuk the method getQueryParams is not suitable for us, since it turns into a string with the saved key if the value is empty. In this case, the key should not be part of the parameter at all.

Reply:
@KatsiarynaTatarynovich You should encrypt `launchName` in URL. So, I advise to use `getQueryParams` but exclude redundant property by the condition.